app/models/concerns/sufia/ability.rb

```diff
@@ -52,6 +52,7 @@ def editor_abilities
 
       can :create, TinymceAsset
       can [:create, :update], ContentBlock
+      can :edit, ::SolrDocument
```

Comment:
The argument against putting admin rights in metadata is imagining you have multiple (N) hydra heads on the same repository.  Distinct domains.  Each head would have its own admins.  If each head is not aware of all others permissions expectations (a maintenance nightmare), then you end up having to have all heads listen for all potentially relevant events and serially inject their own rights strings, versioning your object N-1 additional times.  Massive performance cost and logical complexity. 

An app tracking and constraining its own admin rights is a favor to the rest of the ecosystem.  The rights MD should represent object rights on a timescale beyond the lifespan of an application.